### PR TITLE
test: assign by ref

### PR DESCRIPTION
**Pull request by evilebottnawi**
We should saving test to avoid future regression, it is very bad practice if we don't do it

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Please don't merge failed PRs (linting) :disappointed: 

Maintaining the repository is very terrible, let's stick to standard practices, this solved most of the problems

**Comment by ichiriac** [comment]
I think it's simpler to add `--fix` on linter for automatic fixes on precommit hook

File: test/snapshot/assign.test.js [hunk 1]
```diff
@@ -46,4 +46,7 @@ describe('assign', () => {
   it('>>=', () => {
     expect(parser.parseEval("$var >>= $var;")).toMatchSnapshot();
   });
+  it('with ref', () => {
+    expect(parser.parseEval("$bar = &$foo;")).toMatchSnapshot();
```
Comment (ichiriac) [inline test/snapshot/assign.test.js]: does not test positions

Comment (evilebottnawi) [reply test/snapshot/assign.test.js]: for position we use other test file, i think we should keep position for all tests, but let's do it in other PR

**Pull request merged**